homu/init.sls

```diff
@@ -9,15 +9,21 @@ homu-debugging-packages:
       - sqlite3
 
 homu:
+  user.present:
+    - fullname: Homu
+    - shell: /bin/bash
+    - home: /home/servo/homu
```

Comment:
Homu should have its own home directory completely separate from servo, i.e. `/home/homu`. We'll also need to have some manual deployment steps for this to move the homu db to the right place by hand.